source/extensions/access_loggers/grpc/grpc_access_log_utils.cc

```diff
@@ -203,7 +203,9 @@ void Utility::extractCommonAccessLogProperties(
       peer_san->set_uri(MessageUtil::sanitizeUtf8String(uri_san));
     }
 
-    peer_properties->set_subject(downstream_ssl_connection->subjectPeerCertificate());
+    peer_properties->set_subject(MessageUtil::sanitizeUtf8String(downstream_ssl_connection->subjectPeerCertificate()));
```

Comment:
This sanitization like a very minor behavior change. Is this necessary?

Reply:
removed.